packages/sui-studio/bin/migrations/component-files.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-console */
+
+const glob = require('fast-glob')
+const {resolve, dirname, join} = require('node:path')
+const {readFile, stat, rename, writeFile} = require('node:fs/promises')
+const pattern = /export default(?: function)? ([$A-Z_][0-9A-Z_$]*)/i
+
+const extractComponeName = content => {
+  const match = content.match(pattern)
+  if (!match) return null
+
+  const componentName = match[1]
+
+  return componentName ?? null
+}
+
+const hasMultipleExports = content => {
+  const match = content.match(/^export /gm)
+
+  if (!match) return null
```

Comment:
```suggestion
  if (!match) return false
```

given the "boolean" name of the function... 😉

Reply:
Agree! Thanks for the feedback 🙌